hoot-core/src/main/cpp/hoot/core/io/HootNetworkRequest.h

```diff
@@ -57,6 +58,18 @@ class HootNetworkRequest
   bool networkRequest(QUrl url,
     QNetworkAccessManager::Operation http_op = QNetworkAccessManager::Operation::GetOperation,
     const QByteArray& data = QByteArray());
+  /**
```

Comment:
Considered also making this work for raw headers, as well as known request headers, but since it wasn't needed by anything right now, didn't do it.